app/main.py

```diff
@@ -1,23 +1,34 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from django.db.models import QuerySet, Q, F
+from db.models import LoyaltyProgram, LoyaltyProgramParticipant, Customer
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.all().values_list(
+        "name", "bonus_percentage")
```

Comment:
Please, move the closing parenthesis to the next line when you format code like this.